pkgs/development/tools/confluent-cli/default.nix

```
@@ -37,7 +49,7 @@ stdenv.mkDerivation rec {
     # build it. Get the list of supported system by looking at the list of
     # files in the S3 bucket:
     #
-    #   https://s3-us-west-2.amazonaws.com/confluent.cloud?prefix=confluent-cli/archives/1.25.0/&delimiter=/%27
-    platforms = [ "x86_64-linux" "x86_64-darwin" ];
+    #   https://s3-us-west-2.amazonaws.com/confluent.cloud?prefix=confluent-cli/archives/3.17.0/&delimiter=/%27
+    platforms = [ "x86_64-linux" "x86_64-darwin" "aarch64-darwin" "aarch64-linux"];
```

In this case, couldn't it be 
```suggestion
    platforms = platforms.unix;
```